src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1791,6 +1791,8 @@
     <Field Name="IsDelegateCall" Type="bool"/>
     <Field Name="Expanded" Type="bool"/>
     <Field Name="InvokedAsExtensionMethod" Type="bool"/>
+    <!-- Whether receiver will need to be cloned during emit (only valid before lowering) -->
+    <Field Name="ReceiverCloned" Type="bool"/>
```

Comment:
>ReceiverCloned [](http://example.com/codeflow?start=17&length=14)

We should consider more "scary" name. Also, the name sounds like the cloning has happened already, whereas it is going to happen later. Something like "InitialBindingReceiverIsSubjectToCloning".